### PR TITLE
Tweak the Python API docs to look nicer

### PR DESCRIPTION
**Pull request by emilk**
### What
* Closes https://github.com/rerun-io/rerun/issues/3621

Tweaks the Python API docs to look nicer.

Smaller example images (and fewer examples) reduces the amount of scrolling required to see the `init` method:

image

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested [demo.rerun.io](https://demo.rerun.io/pr/3622) (if applicable)

- [PR Build Summary](https://build.rerun.io/pr/3622)
- [Docs preview](https://rerun.io/preview/99746c8cb755e17638b60dff3e9303ef0301b4db/docs) 
- [Examples preview](https://rerun.io/preview/99746c8cb755e17638b60dff3e9303ef0301b4db/examples) 
- [Recent benchmark results](https://ref.rerun.io/dev/bench/)
- [Wasm size tracking](https://ref.rerun.io/dev/sizes/)

### CODE REVIEW

File: crates/re_types_builder/src/codegen/python.rs [hunk 1]
```diff
@@ -1735,6 +1739,8 @@ fn quote_clear_methods(obj: &Object) -> String {
 // --- Arrow registry code generators ---
 use arrow2::datatypes::{DataType, Field, UnionMode};
 
+use super::common::ExampleInfo;
```
Comment (Wumpf) [inline crates/re_types_builder/src/codegen/python.rs:1742]: strange location for something that is needed way further up

File: rerun_py/rerun_sdk/rerun/archetypes/asset3d_ext.py [hunk 2]
```diff
@@ -27,6 +27,8 @@ def guess_media_type(path: str) -> MediaType | None:
 
 
 class Asset3DExt:
+    """Extension methods for [Asset3D][rerun.archetypes.Asset3D]."""
```
Comment (Wumpf) [inline rerun_py/rerun_sdk/rerun/archetypes/asset3d_ext.py:30]: Can we please add assertion in python.rs that this is present?

File: rerun_py/rerun_sdk/rerun/components/media_type_ext.py [hunk 3]
```diff
@@ -8,6 +8,8 @@
 
 # TODO(#2388): constants in fbs
 class MediaTypeExt:
+    """Extension methods for [MediaType][rerun.components.MediaType]."""
```
Comment (Wumpf) [inline rerun_py/rerun_sdk/rerun/components/media_type_ext.py:11]: Hm so technically there's not a single method in here, only constants 🤔

**Review by Wumpf — APPROVED** [review]
much better!

File: crates/re_types_builder/src/codegen/common.rs [hunk 4]
```diff
@@ -173,14 +173,17 @@ impl RerunImageUrl<'_> {
     pub fn image_stack(&self) -> Vec<String> {
         const WIDTHS: [u16; 4] = [480, 768, 1024, 1200];
 
+        // Don't let the images take up too much space on the page.
+        let desired_with = Some(640);
```
[the remark this answers is not in the document]
Comment (emilk) [reply crates/re_types_builder/src/codegen/common.rs:177]: Depends on your gui scale (pixels per point)

File: crates/re_types_builder/src/codegen/python.rs (hunk 1 above)
Comment (emilk) [reply crates/re_types_builder/src/codegen/python.rs:1742]: rust-analyzer must be trippin'

File: crates/re_types_builder/src/codegen/common.rs (hunk 4 above)
[the remark this answers is not in the document]
Comment (Wumpf) [reply crates/re_types_builder/src/codegen/common.rs:177]: everyone who doesn't either use 100% or 200% clearly lacks taste 😄

**Pull request merged**